Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -p $PORT -C ./config/puma.rb
+worker: bundle exec sidekiq -c 3
```

Comment:
use config as a file and set in the file all what you need (instead of `-c 3 ... etc`)